draft-ietf-webpush-protocol.xml

```diff
@@ -574,24 +607,100 @@ DATA         [stream 4] +END_STREAM
       </t>
+      <section anchor="monitor-set" title="Receiving Push Messages for a Subscription Set">
+        <t>
+          A user agent requests the delivery of new push messages for a collection of
+          push message subscriptions by making a GET request to a push message subscription
+          set resource. The push service does not respond to this request, it instead uses
+          <xref target="RFC7540">HTTP/2 server push</xref> to send the contents of push
+          messages as they are sent by application servers.
+        </t>
+        <t>
+          Each push message is pushed in response to a synthesized GET request.
+          This GET request is made to the push message resource that was created by
+          the push service when the application server requested message delivery.
+          The push service MUST include the link references to the push and receipt
+          subscribe resources in the request.  The response body is the entity body
+          from the most recent request sent to the push resource.
+        </t>
+
+        <figure>
+          <preamble>
+            The following example request is made over HTTP/2.
+          </preamble>
+          <artwork>
+            <![CDATA[
+HEADERS      [stream 7] +END_STREAM +END_HEADERS
+  :method        = GET
+  :path          = /set/4UXwi2Rd7jGS7gp5cuutF8ZldnEuvbOy
+  :authority     = push.example.net
+]]>
+          </artwork>
+        </figure>
+
+        <figure>
+          <preamble>
+            The push service permits the request to remain outstanding.  When a
+            push message is sent by an application server, a server push is
+            associated with the initial request.  The response includes the push
+            message.
+          </preamble>
+          <artwork>
+            <![CDATA[
+PUSH_PROMISE [stream 7; promised stream 4] +END_HEADERS
+  :method        = GET
+  :path          = /d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
+  :authority     = push.example.net
+  :link          = </p/JzLQ3raZJfFBR0aqvOMsLrt54w4rJUsV>;
+                    rel="urn:ietf:params:push"
+  :link          = </receipts/xjTG79I3VuptNWS0DsFu4ihT97aE6UQJ>;
+                    rel="urn:ietf:params:push:receipt"
 
+HEADERS      [stream 4] +END_HEADERS
+  :status        = 200
+  date           = Thu, 11 Dec 2014 23:56:56 GMT
+  last-modified  = Thu, 11 Dec 2014 23:56:55 GMT
+  cache-control  = private
+  content-type   = text/plain;charset=utf8
+  content-length = 36
+
+DATA         [stream 4] +END_STREAM
+  iChYuI3jMzt3ir20P8r_jgRR-dSuN182x7iB
+]]>
+          </artwork>
+        </figure>
+
+
+        <t>
+          A user agent can request the contents of the push message subscription
+          set resource immediately by including a <xref target="RFC7240">Prefer header
+          field</xref> with a "wait" parameter set to "0".
+        </t>
+        <t>
+          In response to this request, the push service MUST generate a server push

```

Comment:
I wouldn't break this paragraph into two.  The "this" is hard to place if you do.